jinahub/indexers/indexer/LMDBIndexer/__init__.py

```diff
@@ -55,10 +54,11 @@ def index(self, docs: DocumentArray, parameters: Dict, **kwargs):
         :param docs: the documents to add
         :param parameters: parameters to the request
         """
-        trav_path = parameters.get('traversal_path', self.default_traversal_path)
+        trav_paths = parameters.get('traversal_paths', self.default_traversal_paths)
         with self.env.begin(write=True) as t:
-            for d in docs.traverse_flat(trav_path):
-                t.put(d.id.encode(), d.SerializeToString())
+            for trav_path in trav_paths:
+                for d in docs.traverse_flat(trav_path):
```

Comment:
This should not work. `traverse_flat` expects `Iterable[str]` and not `str`. The reason it works currently is, that iterating over a string gives back chars. It should be just:
```
for d in docs.traverse_flat(traversal_paths)
```

Reply:
done